.github/workflows/linter.yml

```diff
@@ -0,0 +1,50 @@
+---
+# MegaLinter GitHub Action configuration file
+# More info at https://megalinter.io
+name: MegaLinter
+
+on: # yamllint disable-line rule:truthy
+  # Trigger mega-linter at every push. Action will also be visible from Pull Requests to main
+  pull_request:
+    branches:
+      - develop
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: MegaLinter
+    runs-on: ubuntu-latest
+    steps:
+      # Git Checkout
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.PAT || secrets.GITHUB_TOKEN }}
```

Comment:
I try to avoid PAT, but use app token in stead.

Reply:
Copy & paste. PAT removed as not needed.